sdk/src/index.ts

```diff
@@ -0,0 +1,19 @@
+export * from './Vyper';
+
+
+// for testing
```

Comment:
Can we use a more standard approach for testing? I'd like to introduce [Mocha](https://mochajs.org/), we're using it in all our project. At [this link](https://mochajs.org/#getting-started) you have a getting started guide

Reply:
Seems good, will setup mocha